app/assets/stylesheets/_layout.scss

```diff
@@ -1,3 +1,7 @@
 .content {
-  @include outer-container;
+	 @include outer-container(68em);
+}
+
+.content.high_voltage-pages {
```

Comment:
Merge rule `.content.high_voltage-pages` with rule on line 1<br>Selector `high_voltage-pages` should be written in lowercase with hyphens